File: problemas/1000/1015.cs
using System;

namespace uri1015
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            double x1 = double.Parse(entrada[0]);
            double y1 = double.Parse(entrada[1]);
            entrada = Console.ReadLine().Split(' ');
            double x2 = double.Parse(entrada[0]);
            double y2 = double.Parse(entrada[1]);

            double distancia = Math.Sqrt(Math.Pow(x2 - x1, 2) + Math.Pow(y2 - y1, 2));
            Console.WriteLine(distancia.ToString("F4"));
        }
    }
}

File: problemas/1000/1015.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        var sc = new Scanner(System.in);
        String[] entrada = sc.nextLine().split(" ");
        double x1 = Double.parseDouble(entrada[0]);
        double y1 = Double.parseDouble(entrada[1]);
        entrada = sc.nextLine().split(" ");
        double x2 = Double.parseDouble(entrada[0]);
        double y2 = Double.parseDouble(entrada[1]);

        double distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));
        System.out.printf("%.4f\n", distancia);
    }
}